diagnostic.md

```diff
@@ -23,7 +35,11 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
+//I'm trying to go up to the parent ol, then the parent body, then down to the child form. I am running out of time.
 
+$(document).ready(function(){
```

Comment:
We just want traverse to `<form>` from any `<li>` element.  A few possible ways are `$('li').parent().parent().find('form')`, `$('li').parent().parent().children('form')`, and `$('.listerItem').parent().parent().$('form')`.